browser/brave_content_browser_client.cc

```diff
@@ -128,6 +128,7 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #include "brave/browser/speedreader/speedreader_tab_helper.h"
 #include "brave/components/speedreader/speedreader_throttle.h"
 #include "third_party/blink/public/mojom/loader/resource_load_info.mojom-shared.h"
+using DistillState = speedreader::SpeedreaderTabHelper::DistillState;
```

Comment:
better to move this line down to `CreateURLLoaderThrottles`

Reply:
fixed